pandas/tests/extension/test_arrow.py

```diff
@@ -743,6 +743,13 @@ def test_get_common_dtype(self, dtype, request):
             )
         super().test_get_common_dtype(dtype)
 
+    def test_is_not_string_type(self, dtype):
+        pa_dtype = dtype.pyarrow_dtype
+        if pa.types.is_string(pa_dtype):
```

Comment:
should the binary types be considered stringy?

Reply:
I would say no as of now, as `pa.types.is_string` doesn't consider binary striny

Reply:
fair enough